charts/graylog/values.yaml

```diff
@@ -528,6 +528,10 @@ graylog:
     ringSize: ""
     inputBufferRingSize: ""
 
+  ## The threshold of the garbage collection runs. If GC runs take longer than this
+  ## threshold, a system notification will be generated to warn the administrator
+  ## about possible problems with the system. Default is 1 second.
+  gc_warning_threshold: 1s
```

Comment:
@lkeijser can you move to under `options`? it will be `.Values.graylog.options.gc_warning_threshold`.
So we group graylog config under the same group.

Reply:
Done!